### ISSUE TITLE
Code generation for EFT gg>h : issues with model parameters

### ISSUE THREAD

**Issue by valassi** [madgraph5/madgraph4gpu#351]
Hi @oliviermattelaer this is the second part of the followup of #344.

You suggested to try EFT gg>h (nice idea!). Indeed I had (have) a few minor issues to fix, but now I am also facing a more fundamental issue: I have
```
      // Wavefunction(s) for diagram number 1
      vxxxxx<M_ACCESS, W_ACCESS>( momenta, 0., cHel[ihel][0], -1, w_fp[0], 0 );

      vxxxxx<M_ACCESS, W_ACCESS>( momenta, 0., cHel[ihel][1], -1, w_fp[1], 1 );

      sxxxxx<M_ACCESS, W_ACCESS>( momenta, m_pars->mdl_MH, +1, w_fp[2], 2 );


      // Amplitude(s) for diagram number 1
      VVS3_0<W_ACCESS, A_ACCESS>( w_fp[0], w_fp[1], w_fp[2], cxmake( cIPC[0], cIPC[1] ), &amp_fp[0] );
      jamp_sv[0] += +2.*amp_sv[0];
```

The problem here is that the `m_pars->mdl_MH` should possibly be in constant memory rather than in this instance data memeber. At least, for the SM Z/W masses this is what we have, they end up in the cIPD block I think.

My question: is there something fundamentally different in your base code generation for EFT, which makes it such that the Higgs mass does not end up in cIPD? I will wait for ou to answer rather than attempt a fix on this one.

(Just FYI if you have lost it in my changes: I have also been prototyping a hardcoded constexpr version of all physics parameters, PR #306. This is even faster than constant memory. It is a related, but DIFFERENT story, just teling you for info. In any case we first need to fix the issue above).

Thanks! Andrea

**Comment by valassi** [comment]
Ehm... I tried to generate the SM gg>h, but I get an error, of course because there is no such diagram in the SM...
```
DEBUG:  True [misc.py at line 2192] 
INFO: Checking for minimal orders which gives processes. 
INFO: Please specify coupling orders to bypass this step. 
INFO: Trying process: g g > h WEIGHTED<=2 @1  
Command "import /data/avalassi/GPU2020/MG5aMC/2.7.0_gpu/CODEGEN_cudacpp_gg_h.mg" interrupted in sub-command:
"define vl~ = ve~ vm~ vt~" with error:
NoDiagramException : No amplitudes generated from process Process: g g > h WEIGHTED=2 @1. Please enter a valid process
Please report this bug on https://bugs.launchpad.net/mg5amcnlo
More information is found in 'MG5_debug'.
Please attach this file to your report.
```

**Comment by valassi** [comment]
The issue with model parameters is now clarified (see also #344 and #356): I have removed both th emass and the helicity from sxxxxx. This is in PR #352, which I am about to merge, and will close the issue.

Note also that there was another question #358 about RAMBO for 2>1 proceses like EFT gg>h: this gives a warning, which is a valid warning as 2>1 are special. I will keep the warning, I have closed #358.

I am now closing this one too.
